lib/ansible/module_utils/eos.py

```diff
@@ -40,7 +40,7 @@
 
```

Comment:
Might be worth adding a note here to say *why* defaults are not set here, and direct people to ARGS_DEFAULT_VALUE